website/content/docs/security/acl/auth-methods/aws-iam.mdx

```diff
@@ -0,0 +1,199 @@
+provided pre-signed AWS API requests.
+
+An IAM role or user authenticates by presenting certain signed AWS API requests in a specific JSON
+format. The client signs the necessary AWS API requests with its AWS credentials using the [AWS
+Signature v4 algorithm](https://docs.aws.amazon.com/general/latest/gr/sigv4_signing.html). When the
+auth method receives the signed AWS API requests, it forwards the requests to AWS. AWS validates the
+client's signature and, if the signature is valid, responds with the client's identity details. The
+signature validation performed by AWS on the `sts:GetCallerIdentity` request provides the auth
+method with a strong guarantee of the client's identity. The auth method compares the Amazon
+Resource Name (ARN) of the client with the `BoundIAMPrincipalARNs` list to determine if the client
+is permitted to login.
+
+## Config Parameters
+
+The following are the auth method [`Config`](/api-docs/acl/auth-methods#config)
+parameters for an auth method of type `aws-iam`:
+
+- `BoundIAMPrincipalARNs` `(array<string>: <required>)` - The list of IAM role or IAM user ARNs
+  which are permitted to login. A client authenticating to Consul must have an ARN that matches one
+  of the ARNs in this list.
+   - If `EnableIAMEntityDetails=false`, then bound ARNs must not contain the full path of the role
+     or user, and wildcards are not supported. For example,
+     `arn:aws:iam::123456789012:user/MyUserName` will permit the IAM user named "MyUserName" to log
+     in, and `arn:aws:iam::123456789012:role/MyRoleName` will permit the IAM role named "MyRoleName"
+     to log in.
+   - If `EnableIAMEntityDetails=true`, then bound ARNs with the full path must be used, such as,
+     `arn:aws:iam::123456789012:role/path/to/MyRoleName`. Additionally, ARNs may contain a single
+     trailing wildcard. For example, `arn:aws:iam::123456789012:*` will permit any role or user in
+     the account `123456789012` to login, while `arn:aws:iam::123456789012:role/path/to/roles/*`
+     will permit only roles at the path `/path/to/roles/`.
+- `EnableIAMEntityDetails` `(bool: <false>)` - This enables the auth method to fetch the IAM role or
+  IAM user details, including tags and the full role or user path. If enabled, clients must pass the
+  `-aws-include-entity` option to `consul login`. Additionally, an IAM role or user attempting to
+  login must have `iam:GetRole` or `iam:GetUser` permission, respectively, to retrieve itself. This
+  is required in order to fetch the full path and tags of the IAM user or role, and it is required
+  in order to use wildcards in the `BoundIAMPrincipalARNs`.
+- `IAMEntityTags` `(array<string>: [])` - The list of tag keys retrieved from the IAM role or user
+  and made available to binding rules. Tags are only supported when `EnableIAMEntityDetails=true`.
+  Only the tags in this will be available in binding rules as `entity_tags.<key>`. For example, if
```

Comment:
What is "this" in context to? Should it be "Only tags in roles or users will be avaiable"?

Reply:
Fixed, and expanded/revised this point a bit.